Tests/PlotTests/HTMLComponentTests.swift

```diff
@@ -426,6 +426,14 @@ final class HTMLComponentTests: XCTestCase {
 
         XCTAssertEqual(html, "<ol><li>One</li><li>Two</li></ol>")
     }
+    
+    func testTime() {
+        let html = Time(datetime: "2011-11-18T14:54:39Z") {
+            Paragraph("Hello World")
+        }.render()
```

Comment:
Code style nitpick: Please place this `render` call on the next line for improved readability (as is done elsewhere across this file).